realm/realm-library/src/objectServer/java/io/realm/permissions/PermissionOffer.java

```diff
@@ -94,7 +94,7 @@ public PermissionOffer(String url, AccessLevel accessLevel) {
      * @param accessLevel the {@link AccessLevel} granted to the user accepting the offer.
```

Comment:
This comment is for the line 91:
Creates a request for a permission offer that last until it is manually revoked. -> Creates a request for a permission offer that lasts until it is manually revoked?